src/templates/static/404.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
 const Page404 = () => (
-    <div className='container'>
-        <div className='page_404 static_full'>
+    <div className='container static-page-layout static_full'>
```

Comment:
we are not reskinning the 404 page?

Reply:
Yes @balakrishna-deriv  not in this phase, there's no design figma ready.